ci/repokitteh/modules/newcontributor.star

```diff
@@ -0,0 +1,43 @@
+
+NEW_CONTRIBUTOR_MESSAGE = """
+Hi @%s, welcome and thank you for your contribution.
+
+We will try to review your Pull Request as quickly as possible.
+
+In the meantime, please take a look at the [contribution guidelines](https://github.com/envoyproxy/envoy/blob/main/CONTRIBUTING.md) if you have not done so already.
+
+"""
+
+DRAFT_MESSAGE = """
+As a reminder, PRs marked as draft will not be automatically assigned reviewers,
+or be handled by maintainer-oncall triage.
+
+Please mark your PR as ready when you want it to be reviewed!
+"""
+
+
+def get_pr_author_association(issue_number):
+  return github.call(
```

Comment:
4-space indent

Reply:
This is a file copy so I'm going to let it stay as the original file is.  I'm inclined to say in general that if we care about white-space we should have automated checks rather than spend review/reviewer cycles on it.